internal/statestore/backfill.go

```diff
@@ -143,6 +151,48 @@ func (rb *redisBackend) UpdateBackfill(ctx context.Context, backfill *pb.Backfil
 	return nil
 }
 
+// CleanupBackfills removes expired backfills
+func (rb *redisBackend) CleanupBackfills(ctx context.Context) error {
+	redisConn, err := rb.redisPool.GetContext(ctx)
+	if err != nil {
```

Comment:
use similar shorter error handling pattern as suggested above (also in other places in the code)

Reply:
I suggest leaving this one as is, and in other places in statestore too. If I change it to the shorter pattern you suggest then redisConn won't be visible out of this if-scope. Then I need to create a separate variable for that. So I don't think it's worth changing, we follow this pattern everywhere here. This is a matter of personal preference.

Reply:
Sorry. For methods that return value along with error (like here), it is reasonable to have the error check outside (like you have already). 

For methods that return just err, I think the shorter format is concise and readable. However, a quick check tells me we are using either format in the code - not uniform but I am ok with you not changing the format.